packages/web/components/Player/PlayerAvatar.tsx

```diff
@@ -0,0 +1,29 @@
+import { Avatar, AvatarProps } from '@metafam/ds';
+import React from 'react';
+
+import { PlayerFragmentFragment } from '../../graphql/autogen/types';
+import {
+  getPlayerImage,
+  getPlayerName,
+  hasPlayerImage,
+} from '../../utils/playerHelpers';
+
+type Props = { player: PlayerFragmentFragment; page: string };
+export const PlayerAvatar: React.FC<Props> = ({ player, page }) => {
```

Comment:
Rather than having the conditional on `page`, I'd only use the `src` and `name` attributes, and specify `size`, `w`, & `h` on the pages where they're included.

```suggestion
export const PlayerAvatar: React.FC<Props> = ({ player, ...props }) => {
  props.src ??= getPlayerImage(player);
  props.name ??= getPlayerName(player);
  if (hasPlayerImage(player)) props.bg ??= 'transparent';
  return <Avatar {...props} />;
};
```